backend/src/graphql/resolvers/users/location.spec.ts

```diff
@@ -1,16 +1,30 @@
 /* eslint-disable @typescript-eslint/no-unsafe-assignment */
 /* eslint-disable @typescript-eslint/no-unsafe-member-access */
 /* eslint-disable @typescript-eslint/no-unsafe-call */
-import { createTestClient } from 'apollo-server-testing'
 import gql from 'graphql-tag'
 
 import Factory, { cleanDatabase } from '@db/factories'
-import { getNeode, getDriver } from '@db/neo4j'
-import createServer from '@src/server'
+import type { ApolloTestSetup } from '@root/test/helpers'
+import { createApolloTestSetup } from '@root/test/helpers'
+import type { Context } from '@src/server'
 
-const neode = getNeode()
-const driver = getDriver()
-let authenticatedUser, mutate, query, variables
+import { mapboxResponses } from './location.spec/mapboxResponses'
+
+let authenticatedUser, variables
+// eslint-disable-next-line @typescript-eslint/no-unsafe-return
+const contextUser = () => authenticatedUser
+let mutate: ApolloTestSetup['mutate']
+let query: any // eslint-disable-line @typescript-eslint/no-explicit-any
+let database: ApolloTestSetup['database']
+let server: ApolloTestSetup['server']
+
+const fetchMock: Context['fetch'] = (url) => {
+  const response = mapboxResponses[url] // eslint-disable-line security/detect-object-injection
+  if (!response) {
+    throw new Error(`Missing response for url: ${url}`)
+  }
+  return response // eslint-disable-line @typescript-eslint/no-unsafe-return
+}
```

Comment:
Why context fetch - i really don't get this. It has no side effects. By that logic we must mock `console.log`, `parseInt` and all other functionality?!

Reply:
Oh, I found out that our CI is sending API requests on every build against Mapbox. That's a common bad practice. It slows down CI and also makes the build depend on the availability of an external API.

Reply:
It has nothing to do with this PR and I believe it should not be part of the context to mock it easier.